lib/types.ts

```diff
@@ -1,5 +1,4 @@
 import type {Merge} from 'type-fest';
-import type {BuiltIns} from 'type-fest/source/internal';
```

Comment:
This is not a stable api in type-fest, it was actually moved to another file in next type-fest major version which can lead to a lot of silent type errors in projects using react-native-onyx.